packages/testcontainers/src/containers/RegTestContainer/Masternode.ts

```diff
@@ -210,6 +224,48 @@ export class MasterNodeRegTestContainer extends RegTestContainer {
     }, timeout, 100, 'waitForPriceInvalid')
   }
 
+  /**
+   * Wait for active price
+   *
+   * @param {string} fixedIntervalPriceId
+   * @param {number} activePrice
+   * @param {number} [timeout=30000] in ms
+   * @return {Promise<void>}
+   */
+  async waitForActivePrice (fixedIntervalPriceId: string, activePrice: number, timeout = 100000): Promise<void> {
+    return await waitForCondition(async () => {
+      const data: any = await this.call('getfixedintervalprice', [fixedIntervalPriceId])
+      // eslint-disable-next-line
+      if (data.activePrice !== activePrice) {
+        await this.generate(1)
+        return false
+      } else {
+        return true
+      }
+    }, timeout, 100, 'waitForActivePrice')
+  }
+
+  /**
+   * Wait for next price
+   *
+   * @param {string} fixedIntervalPriceId
+   * @param {number} nextPrice
+   * @param {number} [timeout=30000] in ms
+   * @return {Promise<void>}
+   */
+  async waitForNextPrice (fixedIntervalPriceId: string, nextPrice: number, timeout = 100000): Promise<void> {
```

Comment:
`activePrice`/`nextPrice` should be parsed `string` as they are precision sensitive

Reply:
Ok. I am going to fix this in another PR.

Reply:
Fixed in UpdateVault DFTX PR